real_estate/data/website_view.xml

```diff
@@ -12,7 +12,7 @@
           <t t-set="logged_partner" t-value="request.env['website.visitor']._get_visitor_from_request().partner_id"/>
           <t t-set="contactus_form_values" t-value="{                 'email_to': res_company.email,                 'name': request.params.get('name', ''),                 'phone': request.params.get('phone', ''),                 'email_from': request.params.get('email_from', ''),                 'company': request.params.get('company', ''),                 'subject': request.params.get('subject', ''),             }"/>
           <span class="hidden" data-for="contactus_form" t-att-data-values="contactus_form_values"/>
-          <div id="wrap" class="oe_structure oe_empty">
+          <div id="wrap" class="o_editable oe_structure oe_empty">
```

Comment:
While it goes in the good direction, it doesn't seem to be enough.
When I remove an item from the form and save, the item is back...